app/views/hyrax/dashboard/collections/_edit_actions.html.erb

```diff
@@ -1,8 +1,7 @@
 <h2 class="sr-only"><%= t('hyrax.collection.actions.header') %></h2>
 <div class="actions-controls-collections">
-  <%= link_to t('hyrax.collection.actions.add_works.label'),
+  <%= link_to t('hyrax.collection.actions.add_existing_works.label'),
               hyrax.my_works_path(add_works_to_collection: @form.id),
-              title: t('hyrax.collection.actions.add_works.desc'),
-              class: 'btn btn-default',
-              data: { turbolinks: false } %>
```

Comment:
Did you mean to remove this line? Are we no longer seeing turbolinks-related issues?

Reply:
Nope! Because we are indeed still seeing those failures.  I'm sure this was a copy and paste error I copied from a change that happened before your work.  Glad you caught it.